api/base/views.py

```diff
@@ -393,18 +394,21 @@ def create(self, *args, **kwargs):
 def root(request, format=None, **kwargs):
     """
     The documentation for the Open Science Framework API can be found at [developer.osf.io](https://developer.osf.io).
+    The contents of this endpoint are variable and subject to change without notification.
     """
     if request.user and not request.user.is_anonymous:
         user = request.user
         current_user = UserSerializer(user, context={'request': request}).data
     else:
         current_user = None
+    flags = [item.name for item in Flag.objects.all() if flag_is_active(request, item.name)]
     kwargs = request.parser_context['kwargs']
     return_val = {
         'meta': {
             'message': 'Welcome to the OSF API.',
             'version': request.version,
             'current_user': current_user,
+            'active_flags': flags,
```

Comment:
Are we sure this is something we want to expose to users? We had decided to use the `_` namespace to avoid exposing these.